lib/cli/test/fixtures/react_babel_config_js/package.json

```diff
@@ -15,6 +15,6 @@
     "@babel/plugin-proposal-optional-chaining": "7.0.0",
     "@babel/core": "7.1.0",
     "babel-loader": "8.0.2",
-    "@storybook/react": "^4.0.0-alpha.17"
+    "@storybook/react": "^4.0.0-rc.6"
```

Comment:
It's probably my mistake that those dependencies are in fixtures. They should be added by our CLI in test runs instead